config.py.example

```diff
@@ -1,2 +1,8 @@
 bot_token = '123456:ABCDEFGHI'  # Telegram bot token from @BotFather
 admins_ids = [123, 456]  # List of Telegram user identifiers (integers). Find via @myidbot
+
+# `quick_access_cmds` specifies keyboard buttons that can be used for frequently used commands.
```

Comment:
"that can be used for frequently used"

Damn, didn't I even reread it after I wrote it?